### ISSUE TITLE
Нет статус-кодов ошибок

### ISSUE THREAD

**Issue by IvanKoMaestro** [IT-switcher/verifier#6]
Ни один из запросов раздела "task-resource" не имеет в документации статус-кодов ошибок. Все указанные статусы - из 200 серии; 300,  400, 500 нет.

**Comment by Anastasiya1805** [comment]
Согласна,статусы 300, 400, 500 не задокументированы.ТОлько 200, 201 и 204 тоже нет
